spec/public-api.yaml

```diff
@@ -1,3 +1,4 @@
+# TODO - this is super out of date. Is it more misleading than helpful?
```

Comment:
Good question.

Reply:
Talked to mike about this - we discussed the aspirational goal of organizational/behavioral change to ensure this stays up to date.